src/components/Template/Template.test.tsx

```diff
@@ -1,330 +1,156 @@
+import React from "react";
 import { render, screen } from "@testing-library/react";
 import { axe } from "jest-axe";
-import * as React from "react";
 import renderer from "react-test-renderer";
 
 import {
   Template,
-  TemplateAboveHeader,
-  TemplateHeader,
   TemplateBreakout,
-  TemplateContent,
-  TemplateContentTop,
-  TemplateContentPrimary,
-  TemplateContentSidebar,
-  TemplateFooter,
-  TemplateAppContainer,
+  TemplateTop,
+  TemplateMain,
+  TemplateSidebar,
+  TemplateBottom,
+  SidebarPlacement,
+  TemplateMainNarrow,
 } from "./Template";
 import Placeholder from "../Placeholder/Placeholder";
+import { sidebarLabel } from "../../utils/utils";
 
-const aboveHeader = <Placeholder variant="short">Above Header</Placeholder>;
-const header = <Placeholder variant="short">NYPL Header</Placeholder>;
-const breakout = (
-  <>
-    <Placeholder variant="short">Breadcrumbs</Placeholder>
-    <Placeholder variant="short">Hero</Placeholder>
-  </>
-);
-const sidebar = "left";
+const breakout = <Placeholder variant="short">Breakout</Placeholder>;
 const contentTop = <Placeholder>Content Top</Placeholder>;
-const contentSidebar = <Placeholder>Left Sidebar</Placeholder>;
-const contentPrimary = (
+const contentMain = (
   <>
     <Placeholder>Main Content</Placeholder>
     <Placeholder variant="short">More Content</Placeholder>
   </>
 );
-const footer = <Placeholder variant="short">Footer</Placeholder>;
+const contentBottom = <Placeholder variant="short">Content Bottom</Placeholder>;
+
+const templateComponents = (
+  sidebar: SidebarPlacement = "none",
+  useMainNarrow: boolean = false
+) => (
+  <Template sidebar={sidebar}>
+    <TemplateBreakout>{breakout}</TemplateBreakout>
+    <TemplateTop>{contentTop}</TemplateTop>
+    {sidebar !== "none" && !useMainNarrow && (
```

Comment:
Not too big of a deal since this is a test, but if "sidebar" is "right", the DOM will not be in the right order.